src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -199,8 +199,8 @@
     <script type="text/template" id="template-legend-item-multiple">
         <div class="legend-layer multiple">
             <div class="legend-title"><%- layerName %></div>
-            <% _.each(legend, function(item) { %>
-                <div class="item">
+            <% _.each(legend, function(item, index) { %>
+                <div class="item <% if (index > 4) { %>extra<% } %>">
```

Comment:
One more update to add the count: 
![screen shot 2016-01-19 at 9 33 35 am](https://cloud.githubusercontent.com/assets/1042475/12421341/db7e2ee2-be8f-11e5-9ad3-4da19fdf5a2c.png)